vmdb/db/migrate/20150323003436_add_configured_system_attributes.rb

```diff
@@ -0,0 +1,10 @@
+class AddConfiguredSystemAttributes < ActiveRecord::Migration
+  def change
+    add_column :configured_systems, :ipaddress, :string
+    add_column :configured_systems, :mac_address, :string
+    add_column :configured_systems, :ipmi_present, :boolean
+    add_column :configured_systems, :puppet_status, :string
```

Comment:
@kbrock @brandondunne @gmcculloug I just noticed this PR, so sorry for not commenting sooner, but should this column name have been made more generic?  Something like configuration_status?  Tying the column name to puppet precludes this being used for other things like chef.